### PR TITLE
[HER-49] Create detections query 

### PR DESCRIPTION
**Pull request by Urzica-Platon-Casian**
## Description
Create query with mocked data

## Related
[HER-49](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?selectedIssue=HER-49)

### CODE REVIEW

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]
Not for now! But I need more power. I'll try to get it till tomorrow.

File: src/Presentation/GraphQL/Base/TDetection.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    {
+        public TDetection()
+        {
+            Field(detection => detection.Id).Description("This is the ID");
+            Field(detection => detection.Score).Description("This is the Score");
+            Field(detection => detection.Timestamp).Description("This is the Timestamp");
+            Field(detection => detection.Class).Description("This is the Class");
+        }
+    }
+}
```
Comment (RaduBuciuceanu) [inline src/Presentation/GraphQL/Base/TDetection.cs:19]: Should we move this graph type under the `Detections/Types` feature folder?

File: src/Presentation/GraphQL/Base/Schema.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@ public Schema()
         {
             Mutation = new Mutation();
 
-            Query = new Query();
+            Query = new DetectionQuery();
```
Comment (RaduBuciuceanu) [inline src/Presentation/GraphQL/Base/Schema.cs:13]: Should we maintain the old Query? Should we register the new one as it's child?

File: src/Presentation/GraphQL/Base/DetectionQuery.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    public class DetectionQuery : ObjectGraphType
+    {
+        public DetectionQuery()
+        {
+            Field<TDetection>(
+                "detection",
+                resolve: context => new MDetection { Id = 1, Class = "Pet", Score = 60, Timestamp = DateTime.UtcNow });
+        }
+    }
+}
```
Comment (RaduBuciuceanu) [inline src/Presentation/GraphQL/Base/DetectionQuery.cs:16]: Should we move this folder under the `Detections/Query` file?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]
Here is the power. :smile:

**Review by miutamihai — CHANGES_REQUESTED** [review]
image
Das is no bueno, amigito. Let's remake our commit history so that we remove that ugly `temp` :smile:

**Review by RaduBuciuceanu — APPROVED** [review]

**Review by miutamihai — APPROVED** [review]

**Review by TBiannca — APPROVED** [review]

**Pull request merged**